docs/spec.json

```diff
@@ -0,0 +1,383 @@
+{
+  "name": "corectl",
+  "info": {
+    "title": "Specification for corectl",
+    "description": "corectl contains various commands to interact with the Qlik Associative Engine. See respective command for more information",
+    "version": "v0.4.0",
+    "license": "MIT"
+  },
+  "clispec": "0.1.0",
+  "description": "corectl contains various commands to interact with the Qlik Associative Engine. See respective command for more information",
```

Comment:
No description property allowed on root object

Reply:
added code to be able to remove it 😳
https://598-146282949-gh.circle-artifacts.com/0/home/circleci/corectl/docs/spec.json